cypress/integration/home.spec.js

```diff
@@ -0,0 +1,12 @@
+describe("Home Page", () => {
+  it("has a nav bar", () => {
+    cy.visit("http://localhost:3000");
+
+    // a nav element with class navbar
+    cy.get("nav.navbar").should("exist");
+  });
+  it("has a home page brand button on nav bar", () => {
+    cy.visit("http://localhost:3000");
```

Comment:
Refactoring the `cy.visit` call into a `beforeEach` would probably be much dryer:

https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Hooks

Reply:
done